planning/behavior_velocity_planner/include/utilization/trajectory_utils.hpp

```diff
@@ -142,18 +142,29 @@ inline bool smoothPath(
       0, trajectory.size(), external_v_limit->max_velocity, trajectory);
   }
   const auto traj_lateral_acc_filtered = smoother->applyLateralAccelerationFilter(trajectory);
-  // calc ego internal division point on path
-  const auto traj_with_ego_point_with_idx =
-    getLerpTrajectoryPointWithIdx(*traj_lateral_acc_filtered, current_pose.position);
+  auto nearest_idx =
+    tier4_autoware_utils::findNearestIndex(*traj_lateral_acc_filtered, current_pose.position);
+  const auto dist_to_nearest = tier4_autoware_utils::calcSignedArcLength(
```

Comment:
@TomohitoAndo 
This also looks good but I think calculate distance in world coordinate would be enough why do you use calcSignedArclength?